quiz.js

```diff
@@ -0,0 +1,52 @@
+/* Create a `tree` function that should build a pine tree out of a character in
+the Chrome dev tools console. */
+function tree() {
+
+/* ## It accepts a single object as an argument. The object should have two
+key/value pairs. */
+  var Obj = {
```

Comment:
This object should be created before the `tree()` runs so it can be passed into it.